INN/Largo
# #1540 update grunt dependencies

## Changes

This pull request makes the following changes:

- Update Grunt dependencies to latest versions
- add package.json lock file to keep npm packages locked to our versions we push
- Update change-log with changes
- build process for grunt was changed to allow developers to run separate commands 

## Why

For #1540

## Testing/Questions

Features that this PR affects:

- Development work flow


Questions that need to be answered before merging:



Steps to test this PR:

1.  Just pull down, merge changes, package.json should have latest versions (see this branches package.json)

## Additional information

WP Buddy Client requesting: (if applicable)

- [x] Contributor has read WP Buddy's [GitHub code of conduct](https://github.com/WPBuddy/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] Contributor would like to be mentioned in the release notes as: admin
